Cargo.toml

```diff
@@ -70,6 +82,10 @@ features = [ "derive" ]
 [dependencies.crossterm]
 version = "0.27.0"
 
+[dependencies.locktick]
+version = "0.3"
+optional = true
+
```

Comment:
Not sure if it makes a huge difference, but you can add `default-features=false` here, as we don't use the `std::sync` locks as far as I can tell.

Reply:
The difference would be marginal, and compile-time only; that feature is meant to not be enabled in `nostd` conditions. In addition, since the `std` locks are (these days) more likely to have a better performance than the `parking_lot` ones, we might want to consider using them for some especially "hot" objects in the future, and having this feature around would make it easier to make such swaps for comparison.